src/main/kotlin/com/vk/admstorm/startup/ChangeSshBackendStartup.kt

```diff
@@ -10,44 +10,50 @@ import com.vk.admstorm.notifications.AdmNotification
 object ChangeSshBackendStartup {
     private val LOG = logger<ChangeSshBackendStartup>()
 
-    private const val SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val LEGACY_SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
     private const val SSH_CONFIG_BACKEND = "ssh.config.backend"
+    private const val SSH_NOTIFICATOR_OPTION = "sshOpenSSHChanges"
 
     private fun changeSshConfiguration(configValue: SshConnectionConfigService.Kind) {
         AdvancedSettings.setEnum(SSH_CONFIG_BACKEND, configValue)
     }
 
     fun changeConfigurationProcess(project: Project) {
+        //TODO: Remove it after 2 month
+        if (PropertiesComponent.getInstance(project).getValue(LEGACY_SSH_NOTIFICATOR_OPTION) != null) {
```

Comment:
Why not `isValueSet`?

Reply:
It seems to me that this violates semantics. We want to check whether the property itself exists, not the value. In this case, if such property does not exist, then null is returned

Reply:
`isValueSet`, just checks if the key exists